hq/app/views/header.scala.html

```diff
@@ -37,7 +40,10 @@ <h2 class="header grey-text text-lighten-5">Security HQ</h2>
                             <a href="/iam"><i class="material-icons black-text">vpn_key</i>Credentials audits</a>
                         </li>
                         <li class="main-nav">
-                            <a href="/documentation/"><i class="material-icons black-text">description</i>Documentation</a>
+                            <a href="/snyk"><i class="material-icons black-text">lock_open</i>Snyk</a>
+                        </li>
+                        <li class="main-nav">
+                            <a href="/documentation"><i class="material-icons black-text">description</i>Documentation</a>
```

Comment:
Going to need to put the `/` back here tooo

> /documentation

Reply:
Let me do that :)